modules/KoreBuild.Tasks/GetToolsets.cs

```diff
@@ -97,7 +97,7 @@ private void GetVisualStudio(KoreBuildSettings.VisualStudioToolset vsToolset)
             {
                 if (vsToolset.Required != KoreBuildSettings.RequiredPlatforms.None)
                 {
-                    Log.LogError($"Could not find an installation of Visual Studio that satisifies the specified requirements in {ConfigFile}. " + 
+                    Log.LogError($"Could not find an installation of Visual Studio that satisifies the specified requirements in {ConfigFile}" +
```

Comment:
After thought: to make this new command more visible, you might consider updating the error message to include instructions such as "Execute 'run.ps1 install vs' to update or install VS"